docs/advanced/zigbee/02_improve_network_range_and_stability.md

```diff
@@ -37,6 +37,15 @@ To change the Zigbee channel Zigbee2MQTT uses you have to set the [`channel` in
 ## Interference from other 2.4 GHz devices
 Any device using the open 2.4 GHz spectrum could interfere with Zigbee such as Bluetooth or gaming devices like Logitech “Unifying” or “Lightspeed” or Razer “Hyperspeed Wireless”.
 
+This includes devices that you may not realize _are_ 2.4 GHz. Check anything that is wireless including wireless audio transmitters. 
+[JL Audio JL Link TRX](https://www.crutchfield.com/S-kjc9jL5lfL6/p_136TRX/JL-Audio-JLINK-TRX-High-Fidelity-Audio-Transmitter-Receiver-Kit.html)  is known specifically to cause issues.
+
+Utility companies are known to use zigbee with their "smart meters" but may not advertise them as zigbee. 
+
+Other devices that broadcast zigbee, or modified versions of zigbee are known to cause issues. 
```

Comment:
> or modified versions of zigbee are known to cause issues. 

Could you elaborate on this?

Reply:
This information initially came from puddly. This PR started because I tried all of the "standard" things. 

Here are some examples, but this seems to be a very commercial oriented solution, so not a great of of public information exists, however enough to think its worth mentioning? 

https://www.develcoproducts.com/software/wireless-protocols/zigbee/zigbee-smart-energy/

https://www.reddit.com/r/homeautomation/comments/ol51kl/got_a_new_meter_installed_it_has_a_zigbee_logo_on/

https://www.reddit.com/r/homeassistant/comments/b65pdr/smart_utility_meters_zigbee/

https://www.reddit.com/r/homeautomation/comments/dk5yga/zigbeebased_smart_meter_reading/

Reply:
I understand the smart meter part but meant

> or modified versions of zigbee are known to cause issues. 

What devices are these?

Reply:
Here is an example: Linktap

https://www.link-tap.com .  validation:  https://community.inovelli.com/t/unable-to-reset-switch-or-start-pairing/12073/21?page=2

Linktap uses a modified zigbee, with their own gateway.  Want me to add the specific device example? Full disclosure I have not had issues with my linktap, and I do love their company/product. But that doesn't mean its not causing issues for others.

Reply:
I see, this is fine, thanks!

Reply:
Thank you!